### PR TITLE
Custom Menu Logo

### PR DESCRIPTION
**Pull request by SharmPRO**
Added Custom Menu Logo feature
screen shot 2016-10-18 at 08 24

### CODE REVIEW

File: lib/templates/module/menu.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <ion-header-bar class="bar-positive">
+	<div ng-if="::main.appLogo" ng-show="main.appLogo != ''">
```
Comment (shprink) [inline lib/templates/module/menu.html]: ng-if here should be enough, why do you use ng-show?

**Comment by SharmPRO** [comment]
True...
BTW i have a New cleaner solution with background Image cache and CSS class

**Comment by shprink** [comment]
Thanks for the PR!

We need a way to disable the icon if we want and also we do not want the img to be tracked by git (avoid conflicts if overwritten).

I would have added the icon property under `menu` here https://github.com/shprink/wordpress-hybrid-client/blob/develop/config/config.default.cson#L45-L51

We need to make sure `null` does not display anything (a way to disable it).

For the img itself I would have added it to the `dist` folder and copy it when installing here https://github.com/shprink/wordpress-hybrid-client/blob/develop/scripts/install.js#L10

**Comment by SharmPRO** [comment]
>We need a way to disable the icon if we want and also we do not want the img to be tracked by git (avoid conflicts if overwritten).
No idea how to manage it

>I would have added the icon property under menu here https://github.com/shprink/wordpress-hybrid-client/blob/develop/config/config.default.cson#L45-L51
Agree

>We need to make sure null does not display anything (a way to disable it).
In config.json null will do

>For the img itself I would have added it to the dist folder and copy it in config when installing here https://github.com/shprink/wordpress-hybrid-client/blob/develop/scripts/install.js#L10
Agree

**Comment by shprink** [comment]
I will merge and release a new version as soon as you modify this :) thanks again!

File: scripts/install.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ copy('../dist/config.xml', '../config.xml');
 copy('../dist/config.scss', '../config/config.scss');
 copy('../dist/index.js', '../config/index.js', true);
 copy('../dist/templates', '../config/templates');
+copy('../dist/logo.png', '../img/logo.png', true);
```
Comment (shprink) [inline scripts/install.js]: sorry if it was not clear but if you do not want git to track this file, it need to live in `config`:

```
copy('../dist/logo.png', '../config/logo.png', true);
```

thanks

File: config/config.default.cson [hunk 3]
```diff
@@ -49,6 +49,7 @@
         "state": "public.posts"
         "params": {}
     "exposeAsideWhen": "(min-width:900px)"
+    "logo": "/img/logo.png" # null will disable
```
Comment (shprink) [inline config/config.default.cson]: You need to use the path to the config file `../config/logo.png` (not sure about the relative path but something similar :))

**Comment by shprink** [comment]
Awesome job! thanks very much, will release a version tonight or 2morrow.

**Pull request merged**